phpunit

```diff
@@ -31,14 +31,35 @@ $cmd = array_map('escapeshellarg', $argv);
 $exit = 0;
 
 if (isset($argv[1]) && 'symfony' === $argv[1]) {
+    array_shift($cmd);
+}
+
+$cmd[0] = "php $PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit --colors=always";
+$cmd = str_replace('%', '%%', implode(' ', $cmd)).' %1$s';
+
+$phpIniMatrix = isset($_SERVER['PHP_INI_MATRIX']) ? explode(' ', $_SERVER['PHP_INI_MATRIX']) : array();
+if ($phpIniMatrix) {
+    if ('\\' !== DIRECTORY_SEPARATOR) {
+        echo "Error: PHP_INI_MATRIX is a Windows-only feature.\n";
+        exit(1);
+    }
+
+    $phpDir = dirname(`php -r "echo PHP_BINARY;"`);
+
+    $newCmd = '(SET X=0';
+    foreach ($phpIniMatrix as $iniFile) {
+        $newCmd .= " & copy /Y $phpDir\\$iniFile $phpDir\\php.ini & $cmd & if %%errorlevel%% NEQ 0 SET X=1";
+    }
+    $cmd = $newCmd .= ' & exit %%X%%)';
+}
```

Comment:
Because this would remove some concurrency, thus make the job slower...
But I can add some "echo" to make it clear which ini file is used!